File: XWSMicroservices/api_gateway/src/main/java/com/dislinkt/apigateway/dto/JobOfferDTO.java
package com.dislinkt.apigateway.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class JobOfferDTO {
    public String company;
    public String position;
    public String jobDescription;
    public String dailyActivities;
    public String preconditions;
}

File: dislinkt-app-frontend/src/app/job-offers/job-offer.dto.ts
export class JobOfferDTO{
    company : string;
    position : string;
    jobDescription: string;
    dailyActivities : string;
    preconditions : string;

    constructor(){
        this.company = '';
        this.position = '';
        this.jobDescription = '';
        this.dailyActivities = '';
        this.preconditions = '';
    }
}